apps/fabric/index.js

```diff
@@ -1,5 +1,7 @@
 import {AppRegistry} from 'react-native';
 import App from './src/App';
 import {name as appName} from './app.json';
+import {DotLottieExample} from './src/DotLottieExample';
 
-AppRegistry.registerComponent(appName, () => App);
+// AppRegistry.registerComponent(appName, () => App);
```

Comment:
Do you plan on keeping App around? If so, maybe App and DotLottieExample should be merged before this PR is merged

Reply:
Done.